src/Behat/Testwork/Cli/ConvertConfigCommand.php

```diff
@@ -0,0 +1,111 @@
+use Behat\Testwork\ServiceContainer\Configuration\ConfigurationLoader;
+use Behat\Testwork\ServiceContainer\Exception\ConfigurationLoadingException;
+use PhpParser\BuilderFactory;
+use PhpParser\Node\Stmt\Nop;
+use PhpParser\Node\Stmt\Return_;
+use PhpParser\NodeTraverser;
+use Symfony\Component\Console\Command\Command as BaseCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+final class ConvertConfigCommand extends BaseCommand
+{
+    private OutputInterface $output;
+
+    public function __construct(
+        private ConfigurationLoader $configurationLoader,
+    ) {
+        parent::__construct('convert-config');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->output = $output;
+        $configPath = $this->configurationLoader->getConfigurationFilePath();
+        if (str_ends_with($configPath, '.php')) {
+            throw new ConfigurationLoadingException(sprintf('Configuration file `%s` is already in PHP format', $configPath));
+        }
+
+        $this->output->writeln(['', 'Starting conversion']);
+        $this->convertFile($configPath);
+        $this->output->writeln(['Conversion finished', '']);
+        return 0;
+    }
+
+    private function convertFile(string $filePath): string
+    {
+        $this->output->writeln('Converting configuration file: ' . $filePath);
+
+        $yamlConfig = (array) Yaml::parse(file_get_contents($filePath));
+
+        if (isset($yamlConfig[Config::IMPORTS_SETTING])) {
+            $imports = $yamlConfig[Config::IMPORTS_SETTING];
+            $basePath = rtrim(dirname($filePath), DIRECTORY_SEPARATOR)  . DIRECTORY_SEPARATOR;
+
+            $newImports = [];
+            foreach ($imports as $import) {
+                $import = $basePath . $import;
+                $newImports[] = substr($this->convertFile($import), strlen($basePath));
+            }
+            $yamlConfig[Config::IMPORTS_SETTING] = $newImports;
+        }
+
+        $config = new Config($yamlConfig);
+
+        $factory = new BuilderFactory();
+
+        $expr = $config->toPhpExpr();
+
+        $configStms = [new Return_($expr)];
+
+        $traverser = new NodeTraverser();
+        $useStatementAdder = new UseStatementAdder();
+        $traverser->addVisitor($useStatementAdder);
+        $configStms = $traverser->traverse($configStms);
+
+        $useStmts = [];
+
+        $useStatements = $useStatementAdder->getUseStatements();
+        sort($useStatements);
+        foreach ($useStatements as $useStatement) {
+            $useStmts[] = $factory->use($useStatement)->getNode();
+        }
+        if (count($useStmts) !== 0) {
+            $useStmts[] = new Nop();
+        }
+
+        $stmts = array_merge($useStmts, $configStms);
+        $stmts[] = new Nop();
+
+        // Find the name of the output file
+        $outputFileName = $filePath;
```

Comment:
we use some regular expressions to find the name of the file that will replace the config file